lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -38,16 +38,12 @@ def random
 
       def resolve_stickiness(stickiness, context)
         case stickiness
-        when 'userId'
-          context.user_id
-        when 'sessionId'
-          context.session_id
         when 'random'
           random
         when 'default'
           context.user_id || context.session_id || random
         else
-          nil
+          context.get_by_name(stickiness)
```

Comment:
you might need to do a `begin`/`rescue` for `KeyError` here in the meantime, if you want to keep this behavior in place.

Reply:
I think it's probably just worth waiting for us to settle on fixing the context issue first. Resolving the nil is secondary to getting the correct properties back